sklearn/metrics/tests/test_dist_metrics.py

```diff
@@ -75,6 +65,7 @@ def dist_func(x1, x2, p):
     )
 
 
+@pytest.mark.parametrize("metric", METRICS_DEFAULT_PARAMS)
```

Comment:
Does this work ? it's not tested function (doesn't start with "test") but it's called in a tested function instead.

Reply:
This is unrelated to this PR intend (likely a left over from previous PRs).
I will remove it.